circle.yml

```diff
@@ -0,0 +1,46 @@
+machine:
+  php:
+    version: 5.5.21
+  ruby:
+    version: 2.1.3
+  environment:
+    APP_ENV: testing
+
+general:
+  artifacts:
+    - "tests/codeception/_output"
+    - "storage/logs"
+
+dependencies:
+  cache_directories:
+    - elasticsearch-1.6.0
+    - node_modules
+    - public/components
```

Comment:
well, `~/.npm/` instead of `node_modules`, `~/.composer/cache` instead of `/vendor`